dotnetv3/EC2/Tests/EC2WrapperTests.cs

```diff
@@ -0,0 +1,389 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace EC2Tests;
+/// <summary>
+/// Integration tests for the Amazon Elastic Compute Cloud (Amazon EC2)
+/// Basics scenario.
+/// </summary>
+public class EC2WrapperTests
+{
+    private readonly IConfiguration _configuration;
+    private readonly AmazonEC2Client _client;
+    private readonly EC2Wrapper _ec2Wrapper;
+    private readonly SsmWrapper _ssmWrapper;
+
+    private readonly string _ciderBlock;
+    private readonly string _groupName;
+    private readonly string _groupDescription;
+    private readonly string _keyPairName;
+
+    private static string? _allocationId;
+    private static ArchitectureValues? _architecture;
+    private static string? _associationId;
+    private static string? _ec2InstanceId;
+    private static List<Image>? _images;
+    private static string? _imageId;
+    private static List<InstanceTypeInfo>? _instanceTypes;
+    private static string? _instanceType;
+    private static KeyPair? _keyPair;
+    private static string? _secGroupId;
+    private static string? _tempFileName;
+
+    /// <summary>
+    /// The test class constructor.
+    /// </summary>
+    public EC2WrapperTests()
+    {
+        _configuration = new ConfigurationBuilder()
+            .SetBasePath(Directory.GetCurrentDirectory())
+            .AddJsonFile("testsettings.json") // Load test settings from .json file.
+            .AddJsonFile("testsettings.local.json",
+                true) // Optionally load local settings.
+            .Build();
+
+        _ciderBlock = _configuration["CidrBlock"];
+        _groupName = _configuration["GroupName"];
+        _groupDescription = _configuration["GroupDescription"];
+        _instanceType = _configuration["InstanceType"];
+        _keyPairName = _configuration["KeyPairName"];
+
+        _client = new AmazonEC2Client();
+
+        _ec2Wrapper = new EC2Wrapper(_client);
+        _ssmWrapper = new SsmWrapper(new AmazonSimpleSystemsManagementClient());
+
+    }
+
+    /// <summary>
+    /// Test the creation of an Amazon EC2 key pair.
+    /// </summary>
+    /// <returns></returns>
+    [Fact()]
+    [Order(1)]
+    public async Task CreateKeyPairTest()
+    {
+        _keyPair = await _ec2Wrapper.CreateKeyPair(_keyPairName);
+        Assert.NotNull(_keyPair);
+    }
+
+    /// <summary>
+    /// Test the ability to save the values of an Amazon EC2 key pair to a
+    /// temporary file.
+    /// </summary>
+    [Fact()]
+    [Order(2)]
+    public void SaveKeyPairTest()
+    {
+        _tempFileName = _ec2Wrapper.SaveKeyPair(_keyPair);
+        Assert.NotNull(_tempFileName);
+
```

Comment:
Did the linting process not catch this extra space? If not, we should adjust the linter so it's consistent.

Reply:
It doesn't seem to.